arangod/Aql/QuerySnippet.cpp

```diff
@@ -495,6 +495,14 @@ void QuerySnippet::serializeIntoBuilder(
 
       TRI_ASSERT(!remoteParent->hasDependency());
       remoteParent->addDependency(prototypeConsumer);
+    } else {
+      // TODO: Refactor the code above so that we don't copy and paste
```

Comment:
Ah yes this correct now.

If there is no remote => there is no DistributeConsumer which would need the entry of distId.
However the CloneWorker requires it in constructor.

So we can either do this.
Or doe some more complex stuff to make the CloneWorker work without.
I actually prefer this solution for now.